dashboards/k8s-storage-resources-glusterfs-pv.libsonnet

```diff
@@ -0,0 +1,291 @@
+        uid=($._config.grafanaDashboardIDs['k8s-storage-resources-glusterfs-pv.json']),
+      )
+      .addTemplate('Volume', 'gluster_volume_up', 'volume')
+      .addRow(
+        (g.row('Volume Summary') +
+         {
+           showTitle: false,
+           panels: self._panels,
+         })
+        .addPanel(
+          g.panel('Status') +
+          statPanelArgs +
+          g.statPanel('gluster_volume_up{volume = "$Volume"}',) +
+          {
+            valueMaps: [
+              {
+                op: '=',
+                text: 'DOWN',
+                value: '0',
+              },
+              {
+                value: '1',
+                op: '=',
+                text: 'UP',
+              },
+            ],
+            thresholds: '0.9,1.1',
+            colors: [
+              '#cc0000',
+              '#92d400',
+              '#FFFFFF',
+            ],
+            colorValue: true,
+          }
+        )
+        .addPanel(
+          g.panel('Capacity Utilization') +
+          statPanelArgs +
+          g.statPanel('gluster:volume_capacity_used_bytes_total:sum%s / gluster:volume_capacity_total_bytes:sum%s' % [queryLabels, queryLabels])
+        )
+        .addPanel(
+          g.panel('Available Capacity') +
+          statPanelArgs +
+          g.statPanel('gluster:volume_capacity_total_bytes:sum%s - gluster:volume_capacity_used_bytes_total:sum%s' % [queryLabels, queryLabels], 'decbytes')
+        )
+        .addPanel(
+          g.panel('IOPS') +
+          statPanelArgs +
+          g.statPanel('(sum (gluster_volume_profile_total_reads_interval%s) + sum(gluster_volume_profile_total_reads_interval%s))/max(gluster_volume_profile_duration_secs_interval%s)' % [queryLabels, queryLabels, queryLabels], 'none')
+        )
+        .addPanel(
+          g.panel('Capacity Utilization') +
+          graphPanelArgs +
+          g.queryPanel('gluster:volume_capacity_used_bytes_total:sum%s' % queryLabels, '{{$Volume}}') +
+          g.stack +
+          { yaxes: g.yaxes('decbytes') },
+        )
+        .addPanel(
+          g.panel('IOPS') +
+          {
+            span: 6,
+            legend+: { show: false },
+          } +
+          g.queryPanel('(sum (gluster_volume_profile_total_reads_interval%s) + sum(gluster_volume_profile_total_reads_interval%s))/max(gluster_volume_profile_duration_secs_interval%s)' % [queryLabels, queryLabels, queryLabels], '{{$Volume}}') +
+          g.stack
+        )
+      )
+      .addRow(
+        g.row('LVM Summary')
+        .addPanel(
+          g.panel('LVM Thin Pool Metadata %') +
+          g.queryPanel('100*(gluster_thinpool_metadata_used_bytes%s / gluster_thinpool_metadata_total_bytes%s)' % [queryLabels, queryLabels], '{{thinpool_name}}')
+        )
+        .addPanel(
+          g.panel('LVM Thin Pool Data Usage  %') +
+          g.queryPanel('100*(gluster_thinpool_data_used_bytes%s / gluster_thinpool_data_total_bytes%s)' % [queryLabels, queryLabels], '{{thinpool_name}}')
```

Comment:
Same as above

Reply:
Here it is different. Query panel takes absolute values. So this had to be multiplied.